src/dev-starter.ts

```diff
@@ -84,6 +90,18 @@ export class DevStarter {
     });
   }
 
+  private getCurrentWebpackMajorVersion(): Promise<number> {
+    return new Promise((resolve) => {
+      const webpackVersion = spawn('node_modules/.bin/webpack', ['-v']);
+      webpackVersion.stdout.on('data', (data) => {
+        const output: string = data.toString();
+        const matches = output.match(/\d+\.\d+\.\d+/gm);
+        const majorVersion = matches[0].match(/\d+/gm)[0];
+        resolve(Number.parseInt(majorVersion));
```

Comment:
I think the regex could be simplified by using a capture group (which somehow only seems to work without the global flag, but it still matches at any position in the string):
```suggestion
        const matches = output.match(/(\d+)\.\d+\.\d+/);
        const majorVersion = Number.parseInt(matches[1]);
        resolve(majorVersion);
```